### PR TITLE
feat: map of china added with mapchart

### PR DESCRIPTION
**Pull request by qinjie1314520**
This PR adds a new map of China to mapchart

## Description
Add China map to mapchart and provide more detailed description of map data fields, as shown in the figure below
图片

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
- Manual

### Dev activity
- [X] I My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/widgets/MapChartWidget/constants.ts [hunk 1]
```diff
@@ -1041,3 +1042,210 @@ export const dataSetForUSA = [
   { id: "WV", value: "1.30" },
   { id: "WY", value: "2.97" },
 ];
+
+export const dataSetForChina = [
```
Comment (somangshu) [inline app/client/src/widgets/MapChartWidget/constants.ts:1046]: We should shorten this data set and minify it as well maybe! This is increasing the build size for no reason!

**Review by somangshu — CHANGES_REQUESTED** [review]

**Review by somangshu — APPROVED** [review]
LGTM

**Comment by qinjie1314520** [comment]
I don't think I have anything to change. Maybe there are other things to change? Because there is "Some checks were not successful", this is my first time to contribute code, and I feel a little confused

**Comment by somangshu** [comment]
@qinjie1314520 this is not related probably, I am trying to run the CI workflow, Ill let you know if there is anything related to your change!

**Comment by qinjie1314520** [comment]
I see. Thank you very much for your help. Thank you

**Pull request closed without merging**